python-lib/api_designer_utils/utils.py

```diff
@@ -172,3 +188,11 @@ def get_codeenv_output_msg(plugin, env_name):
     libraries_to_install = plugin.get_file(constants.SPEC_PATH).read().decode('utf-8')
     return "The code env {} has been successfully created with following packages : \n {}.".format(
         env_name, libraries_to_install)
+
+
+def is_user_admin():
+    try:
+        _ = dataiku.api_client().list_code_envs()
+        return True
+    except DataikuException:
+        return False
```

Comment:
log the error, never swallow it, in case that the api call fails for another reason.

Reply:
Fixed in fe49f82